frontend/models/product.ts

```diff
@@ -15,17 +16,21 @@ import {
 } from '@lib/shopify-storefront-sdk';
 import { z } from 'zod';
 import shuffle from 'lodash/shuffle';
+import { IFixitAPIClient } from '@ifixit/ifixit-api-client';
 
 export type Product = NonNullable<Awaited<ReturnType<typeof findProduct>>>;
 export type ProductVariant = ReturnType<typeof getVariants>[0];
 export type ProductImage = ReturnType<typeof getFormattedImages>[0];
 
 export async function findProduct(shop: ShopCredentials, handle: string) {
    const storefront = getShopifyStorefrontSdk(shop);
+   const ifixitApiClient = new IFixitAPIClient({ origin: IFIXIT_ORIGIN });
```

Comment:
This makes 3 different capitalizations of iFixit in this file (IFixitAPIClient ifixitApiClient iFixitProductId)

Reply:
Oh dear. Three worlds collide. :boom: 

I think two is probably unavoidable because types / variables, but three is a bit much.

Reply:
There's actually 4!  `IFIXIT_ORIGIN`